{"error":"D:\\InstagramDashboard\\backend\\src\\models\\Post.js:189\nPostSchema.index({ user_id: 1, instagram_data.posted_at: -1 });\n                                             ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (node:internal/modules/cjs/loader:1512:18)\n    at Module._compile (node:internal/modules/cjs/loader:1534:20)\n    at Object..js (node:internal/modules/cjs/loader:1699:10)\n    at Module.load (node:internal/modules/cjs/loader:1313:32)\n    at Function._load (node:internal/modules/cjs/loader:1123:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)\n    at Module.require (node:internal/modules/cjs/loader:1335:12)\n    at require (node:internal/modules/helpers:136:16)\n    at Object.<anonymous> (D:\\InstagramDashboard\\backend\\src\\services\\DatabaseService.js:2:14)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: Unexpected token '.'\u001b[39m","timestamp":"2025-09-25 02:17:44:1744"}
{"error":"D:\\InstagramDashboard\\backend\\src\\models\\Post.js:189\nPostSchema.index({ user_id: 1, instagram_data.posted_at: -1 });\n                                             ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (node:internal/modules/cjs/loader:1512:18)\n    at Module._compile (node:internal/modules/cjs/loader:1534:20)\n    at Object..js (node:internal/modules/cjs/loader:1699:10)\n    at Module.load (node:internal/modules/cjs/loader:1313:32)\n    at Function._load (node:internal/modules/cjs/loader:1123:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)\n    at Module.require (node:internal/modules/cjs/loader:1335:12)\n    at require (node:internal/modules/helpers:136:16)\n    at Object.<anonymous> (D:\\InstagramDashboard\\backend\\src\\services\\DatabaseService.js:2:14)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: Unexpected token '.'\u001b[39m","timestamp":"2025-09-25 02:18:00:180"}
{"error":"D:\\InstagramDashboard\\backend\\src\\models\\Post.js:189\nPostSchema.index({ user_id: 1, instagram_data.posted_at: -1 });\n                                             ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (node:internal/modules/cjs/loader:1512:18)\n    at Module._compile (node:internal/modules/cjs/loader:1534:20)\n    at Object..js (node:internal/modules/cjs/loader:1699:10)\n    at Module.load (node:internal/modules/cjs/loader:1313:32)\n    at Function._load (node:internal/modules/cjs/loader:1123:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)\n    at Module.require (node:internal/modules/cjs/loader:1335:12)\n    at require (node:internal/modules/helpers:136:16)\n    at Object.<anonymous> (D:\\InstagramDashboard\\backend\\src\\services\\DatabaseService.js:2:14)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: Unexpected token '.'\u001b[39m","timestamp":"2025-09-25 02:18:12:1812"}
{"error":"D:\\InstagramDashboard\\backend\\src\\models\\Post.js:189\nPostSchema.index({ user_id: 1, instagram_data.posted_at: -1 });\n                                             ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (node:internal/modules/cjs/loader:1512:18)\n    at Module._compile (node:internal/modules/cjs/loader:1534:20)\n    at Object..js (node:internal/modules/cjs/loader:1699:10)\n    at Module.load (node:internal/modules/cjs/loader:1313:32)\n    at Function._load (node:internal/modules/cjs/loader:1123:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)\n    at Module.require (node:internal/modules/cjs/loader:1335:12)\n    at require (node:internal/modules/helpers:136:16)\n    at Object.<anonymous> (D:\\InstagramDashboard\\backend\\src\\services\\DatabaseService.js:2:14)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: Unexpected token '.'\u001b[39m","timestamp":"2025-09-25 02:19:53:1953"}
{"attempt":1,"error":"URI malformed","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:20:11:2011"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError creating indexes: Cannot call `users.createIndex()` before initial connection is complete if `bufferCommands = false`. Make sure you `await mongoose.connect()` if you have `bufferCommands = false`.\u001b[39m","stack":"MongooseError: Cannot call `users.createIndex()` before initial connection is complete if `bufferCommands = false`. Make sure you `await mongoose.connect()` if you have `bufferCommands = false`.\n    at NativeCollection.<computed> [as createIndex] (D:\\InstagramDashboard\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:237:15)\n    at DatabaseConnection.createIndexes (D:\\InstagramDashboard\\backend\\config\\database.js:177:33)\n    at DatabaseConnection.initialize (D:\\InstagramDashboard\\backend\\config\\database.js:158:18)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:235:26)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-25 02:20:11:2011"}
{"attempt":2,"error":"URI malformed","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:20:16:2016"}
{"attempt":3,"error":"URI malformed","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:20:21:2021"}
{"attempt":4,"error":"URI malformed","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:20:26:2026"}
{"attempt":5,"error":"URI malformed","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:20:31:2031"}
{"attempt":1,"error":"option buffermaxentries is not supported","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:22:40:2240"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError creating indexes: Cannot call `users.createIndex()` before initial connection is complete if `bufferCommands = false`. Make sure you `await mongoose.connect()` if you have `bufferCommands = false`.\u001b[39m","stack":"MongooseError: Cannot call `users.createIndex()` before initial connection is complete if `bufferCommands = false`. Make sure you `await mongoose.connect()` if you have `bufferCommands = false`.\n    at NativeCollection.<computed> [as createIndex] (D:\\InstagramDashboard\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:237:15)\n    at DatabaseConnection.createIndexes (D:\\InstagramDashboard\\backend\\config\\database.js:177:33)\n    at DatabaseConnection.initialize (D:\\InstagramDashboard\\backend\\config\\database.js:158:18)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:235:26)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-25 02:22:40:2240"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /favicon.ico - Route /favicon.ico not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 02:22:40:2240","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"attempt":2,"error":"option buffermaxentries is not supported","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:22:45:2245"}
{"attempt":3,"error":"option buffermaxentries is not supported","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:22:50:2250"}
{"attempt":4,"error":"option buffermaxentries is not supported","level":"\u001b[31merror\u001b[39m","maxAttempts":5,"message":"\u001b[31mMongoDB connection failed:\u001b[39m","timestamp":"2025-09-25 02:22:55:2255"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET / - Route / not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 02:23:26:2326","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/:theboyfrom_maharashtra - Route /api/:theboyfrom_maharashtra not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 02:24:41:2441","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/:theboyfrom_maharashtra - Validation failed: Username can only contain letters, numbers, dots, and underscores\u001b[39m","params":{},"query":{},"statusCode":400,"timestamp":"2025-09-25 02:26:58:2658","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 02:39:06:396"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 02:39:30:3930"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 02:56:00:560"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-25 02:56:53:5653"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 02:56:53:5653"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserAnalytics for ae.vivekk:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 03:03:17:317"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/ae.vivekk/analytics - User @ae.vivekk not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 03:03:17:317","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/ae.vivekk/anlytics - Route /api/user/ae.vivekk/anlytics not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 03:07:16:716","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET / - Route / not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 03:08:14:814","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /favicon.ico - Route /favicon.ico not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:36:46:3646","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET / - Route / not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:39:51:3951","userAgent":"curl/8.2.1"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 10:40:56:4056"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/users - Route /api/users not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:44:25:4425","userAgent":"curl/8.2.1"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/users - Route /api/users not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:44:38:4438","userAgent":"curl/8.2.1"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/* - Route /api/* not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:46:02:462","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/* - Route /api/* not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:46:04:464","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 10:47:00:470"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 10:53:02:532"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboard\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboard\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-25 10:53:50:5350"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @theyboyfrom_maharashtra:\u001b[39m","timestamp":"2025-09-25 10:57:20:5720"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @theyboyfrom_maharashtra:\u001b[39m","timestamp":"2025-09-25 10:57:20:5720"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:20:5720"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:20:5720"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/theyboyfrom_maharashtra - Instagram user not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:57:20:5720","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:20:5720"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/theyboyfrom_maharashtra/posts - User @theyboyfrom_maharashtra not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:57:20:5720","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @theyboyfrom_maharashtra:\u001b[39m","timestamp":"2025-09-25 10:57:21:5721"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @theyboyfrom_maharashtra:\u001b[39m","timestamp":"2025-09-25 10:57:21:5721"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:21:5721"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:21:5721"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/theyboyfrom_maharashtra - Instagram user not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:57:21:5721","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts for theyboyfrom_maharashtra:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:57:21:5721"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/theyboyfrom_maharashtra/posts - User @theyboyfrom_maharashtra not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:57:21:5721","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/vedikaanawkar - Route /api/vedikaanawkar not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:59:24:5924","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @vedikaanawkar:\u001b[39m","timestamp":"2025-09-25 10:59:39:5939"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @vedikaanawkar:\u001b[39m","timestamp":"2025-09-25 10:59:39:5939"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for vedikaanawkar:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:59:39:5939"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for vedikaanawkar:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 10:59:39:5939"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/vedikaanawkar - Instagram user not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 10:59:39:5939","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @mani_dhulipalla:\u001b[39m","timestamp":"2025-09-25 11:07:54:754"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @mani_dhulipalla:\u001b[39m","timestamp":"2025-09-25 11:07:54:754"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for mani_dhulipalla:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:07:54:754"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for mani_dhulipalla:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:07:54:754"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/mani_dhulipalla - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:07:54:754","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @its_rkdwivedi:\u001b[39m","timestamp":"2025-09-25 11:09:16:916"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @its_rkdwivedi:\u001b[39m","timestamp":"2025-09-25 11:09:16:916"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for its_rkdwivedi:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:09:16:916"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for its_rkdwivedi:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:09:16:916"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/its_rkdwivedi - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:09:16:916","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @its_rkdwivedi6:\u001b[39m","timestamp":"2025-09-25 11:10:25:1025"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @its_rkdwivedi6:\u001b[39m","timestamp":"2025-09-25 11:10:25:1025"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for its_rkdwivedi6:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 11:10:25:1025"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for its_rkdwivedi6:\u001b[39m","statusCode":404,"timestamp":"2025-09-25 11:10:25:1025"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/its_rkdwivedi6 - Instagram user not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 11:10:25:1025","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/ajaysarode_patil/post - Route /api/user/ajaysarode_patil/post not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 11:12:35:1235","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @mani_dhulipalla:\u001b[39m","timestamp":"2025-09-25 11:16:36:1636"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @mani_dhulipalla:\u001b[39m","timestamp":"2025-09-25 11:16:36:1636"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for mani_dhulipalla:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:16:36:1636"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for mani_dhulipalla:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:16:36:1636"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/mani_dhulipalla - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:16:36:1636","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:17:51:1751"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:17:51:1751"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:17:51:1751"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:17:51:1751"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/pomegranates_farming - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:17:51:1751","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:18:08:188"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:18:08:188"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:18:08:188"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:18:08:188"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/pomegranates_farming - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:18:08:188","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching profile for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:18:55:1855"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError fetching complete data for @pomegranates_farming:\u001b[39m","timestamp":"2025-09-25 11:18:55:1855"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to refresh data for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:18:55:1855"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for pomegranates_farming:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 11:18:55:1855"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/pomegranates_farming - Instagram service temporarily unavailable\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 11:18:55:1855","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/pomegranates_farming/anlytics - Route /api/user/pomegranates_farming/anlytics not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-25 11:37:26:3726","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"DatabaseService","errorLabelSet":{},"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError finding user virat.kohli:\u001b[39m","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":{"$oid":"7fffffff00000000000001a6"},"maxSetVersion":227,"servers":{"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8328045,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8328456,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8328218,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"}},"setName":"atlas-9fj64e-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"timestamp":"2025-09-25 12:42:48:4248"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for virat.kohli:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 12:42:48:4248"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/virat.kohli - Database error while finding user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 12:42:48:4248","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"DatabaseService","errorLabelSet":{},"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError finding user virat.kohli:\u001b[39m","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":{"$oid":"7fffffff00000000000001a6"},"maxSetVersion":227,"servers":{"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8351868,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8351868,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8351869,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"}},"setName":"atlas-9fj64e-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"timestamp":"2025-09-25 12:43:06:436"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for virat.kohli:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 12:43:06:436"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/virat.kohli - Database error while finding user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 12:43:06:436","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"DatabaseService","errorLabelSet":{},"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError finding user pran_1306:\u001b[39m","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":{"$oid":"7fffffff00000000000001a6"},"maxSetVersion":227,"servers":{"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-00.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8528712,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-01.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8528742,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"},"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017":{"$clusterTime":null,"address":"ac-oplksvs-shard-00-02.kwko2vu.mongodb.net:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":8528712,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"}},"setName":"atlas-9fj64e-shard-0","stale":false,"type":"ReplicaSetNoPrimary"},"timestamp":"2025-09-25 12:46:13:4613"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for pran_1306:\u001b[39m","statusCode":500,"timestamp":"2025-09-25 12:46:13:4613"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/pran_1306 - Database error while finding user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-25 12:46:13:4613","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboard\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboard\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboard\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-25 18:36:12:3612"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::5000\u001b[39m","port":5000,"stack":"Error: listen EADDRINUSE: address already in use :::5000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboardAssignment\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboardAssignment\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-26 08:52:00:520"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /favicon.ico - Route /favicon.ico not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 08:52:56:5256","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/theboyfrom_maharashtra - Route /api/theboyfrom_maharashtra not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 08:54:05:545","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1937:16)\n    at listenInCluster (node:net:1994:12)\n    at Server.listen (node:net:2099:7)\n    at Function.listen (D:\\InstagramDashboardAssignment\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at Server.start (D:\\InstagramDashboardAssignment\\backend\\server.js:238:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-26 08:56:57:5657"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserAnalytics for ae.vivekk:\u001b[39m","statusCode":404,"timestamp":"2025-09-26 09:05:28:528"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/ae.vivekk/analytics - User @ae.vivekk not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 09:05:28:528","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts for therock:\u001b[39m","statusCode":404,"timestamp":"2025-09-26 09:57:25:5725"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/therock/posts?limit=12 - User @therock not found in database\u001b[39m","params":{},"query":{"limit":12},"statusCode":404,"timestamp":"2025-09-26 09:57:25:5725","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserAnalytics for therock:\u001b[39m","statusCode":404,"timestamp":"2025-09-26 09:57:25:5725"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/therock/analytics - User @therock not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 09:57:25:5725","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserAnalytics for kyliejenner:\u001b[39m","statusCode":404,"timestamp":"2025-09-26 10:07:29:729"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts for kyliejenner:\u001b[39m","statusCode":404,"timestamp":"2025-09-26 10:07:29:729"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/kyliejenner/posts?limit=12 - User @kyliejenner not found in database\u001b[39m","params":{},"query":{"limit":12},"statusCode":404,"timestamp":"2025-09-26 10:07:29:729","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/kyliejenner/analytics - User @kyliejenner not found in database\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 10:07:29:729","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-26 10:13:57:1357"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-26 10:13:57:1357"}
{"error":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","timestamp":"2025-09-26 10:13:57:1357"}
{"code":"EPIPE","errno":-4047,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: EPIPE: broken pipe, write\u001b[39m","stack":"Error: EPIPE: broken pipe, write\n    at Socket._write (node:internal/net:63:18)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Console.log (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston\\lib\\winston\\transports\\console.js:87:23)\n    at Console._write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\winston-transport\\modern.js:103:17)\n    at doWrite (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:390:139)\n    at writeOrBuffer (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:381:5)\n    at Writable.write (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_writable.js:302:11)\n    at DerivedLogger.ondata (D:\\InstagramDashboardAssignment\\backend\\node_modules\\readable-stream\\lib\\_stream_readable.js:629:20)","syscall":"write","timestamp":"2025-09-26 10:13:57:1357"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /favicon.ico - Route /favicon.ico not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 10:32:07:327","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /user/ae.vivekk - Route /user/ae.vivekk not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 10:34:23:3423","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/akshitaraishrivastava/refresh - Route /api/user/akshitaraishrivastava/refresh not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-26 11:24:50:2450","userAgent":"PostmanRuntime/7.47.1"}
{"error":"Error: Cannot find module 'sequelize'\nRequire stack:\n- D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\config\\database.js\n- D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js\n    at Function._resolveFilename (node:internal/modules/cjs/loader:1401:15)\n    at defaultResolveImpl (node:internal/modules/cjs/loader:1057:19)\n    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1062:22)\n    at Function._load (node:internal/modules/cjs/loader:1211:37)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.require (node:internal/modules/cjs/loader:1487:12)\n    at require (node:internal/modules/helpers:135:16)\n    at Object.<anonymous> (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\config\\database.js:1:23)\n    at Module._compile (node:internal/modules/cjs/loader:1730:14)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: Cannot find module 'sequelize'\u001b[39m\n\u001b[31mRequire stack:\u001b[39m\n\u001b[31m- D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\config\\database.js\u001b[39m\n\u001b[31m- D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js\u001b[39m","timestamp":"2025-09-28 13:09:51:951"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:15:07:157"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:19:37:1937"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:28:54:2854"}
{"error":"ReferenceError: connect is not defined\n    at Object.<anonymous> (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\config\\database.js:54:31)\n    at Module._compile (node:internal/modules/cjs/loader:1730:14)\n    at Object..js (node:internal/modules/cjs/loader:1895:10)\n    at Module.load (node:internal/modules/cjs/loader:1465:32)\n    at Function._load (node:internal/modules/cjs/loader:1282:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.require (node:internal/modules/cjs/loader:1487:12)\n    at require (node:internal/modules/helpers:135:16)\n    at Object.<anonymous> (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:12:22)","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mUncaught Exception: connect is not defined\u001b[39m","timestamp":"2025-09-28 13:30:17:3017"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.thzrqreuyhjpkvohcexl.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.thzrqreuyhjpkvohcexl.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:32:47:3247"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:37:07:377"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 13:39:35:3935"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 14:01:38:138"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\u001b[39m","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"db.gynfirqnqnckoztdcocf.supabase.co","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND db.gynfirqnqnckoztdcocf.supabase.co\n    at Client._connectionCallback (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorEvent (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\pg\\lib\\connection.js:57:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-28 14:33:28:3328"}
{"body":{},"ip":"::ffff:127.0.0.1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/instagram/tantrafiesta/profile - Route /api/instagram/tantrafiesta/profile not found\u001b[39m","params":{},"query":{},"statusCode":404,"timestamp":"2025-09-28 14:39:30:3930","userAgent":"PostmanRuntime/7.48.0"}
{"context":"DatabaseService","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts:\u001b[39m","path":"PostAnalysis","timestamp":"2025-09-28 16:29:41:2941"}
{"context":"UserController","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUserPosts for tantrafiesta:\u001b[39m","statusCode":500,"timestamp":"2025-09-28 16:29:41:2941"}
{"body":{},"ip":"::ffff:127.0.0.1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta/posts - Database error while retrieving user posts\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-28 16:29:41:2941","userAgent":"PostmanRuntime/7.48.0"}
{"$clusterTime":{"clusterTime":{"$timestamp":"7555149590253535240"},"signature":{"hash":"1XMQihGDOCYwnZwjq346CkC1coI=","keyId":{"high":1750104346,"low":10,"unsigned":false}}},"code":11000,"codeName":"DuplicateKey","errorLabelSet":{},"errorResponse":{"$clusterTime":{"clusterTime":{"$timestamp":"7555149590253535240"},"signature":{"hash":"1XMQihGDOCYwnZwjq346CkC1coI=","keyId":{"high":1750104346,"low":10,"unsigned":false}}},"code":11000,"codeName":"DuplicateKey","errmsg":"Index build failed: a01f4ebc-7cbb-4ea2-a0ba-93baafd8caa2: Collection test.posts ( f6162199-faff-4a39-ad25-3943f2afdf81 ) :: caused by :: E11000 duplicate key error collection: test.posts index: post_id_1 dup key: { post_id: null }","keyPattern":{"post_id":1},"keyValue":{"post_id":null},"ok":0,"operationTime":{"$timestamp":"7555149590253535240"}},"keyPattern":{"post_id":1},"keyValue":{"post_id":null},"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to start server: Index build failed: a01f4ebc-7cbb-4ea2-a0ba-93baafd8caa2: Collection test.posts ( f6162199-faff-4a39-ad25-3943f2afdf81 ) :: caused by :: E11000 duplicate key error collection: test.posts index: post_id_1 dup key: { post_id: null }\u001b[39m","ok":0,"operationTime":{"$timestamp":"7555149590253535240"},"stack":"MongoServerError: Index build failed: a01f4ebc-7cbb-4ea2-a0ba-93baafd8caa2: Collection test.posts ( f6162199-faff-4a39-ad25-3943f2afdf81 ) :: caused by :: E11000 duplicate key error collection: test.posts index: post_id_1 dup key: { post_id: null }\n    at Connection.sendCommand (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:305:27)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:333:26)\n    at async Server.command (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:171:29)\n    at async CreateIndexesOperation.executeCommand (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\operations\\command.js:76:16)\n    at async CreateIndexesOperation.execute (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\operations\\indexes.js:122:9)\n    at async tryOperation (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\operations\\execute_operation.js:207:20)\n    at async executeOperation (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\operations\\execute_operation.js:75:16)\n    at async Collection.createIndex (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\mongodb\\lib\\collection.js:329:25)\n    at async Object.initialize (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\config\\database.js:58:5)","timestamp":"2025-09-28 20:09:02:92"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Cannot read properties of undefined (reading 'databaseService')\u001b[39m","stack":"TypeError: Cannot read properties of undefined (reading 'databaseService')\n    at getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:19:42)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at validateUsername (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\routes\\userRoutes.js:17:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:284:15\n    at param (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:365:14)","timestamp":"2025-09-28 20:09:40:940"}
{"body":{},"ip":"::ffff:127.0.0.1","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta/ - Cannot read properties of undefined (reading 'databaseService')\u001b[39m","params":{},"query":{},"timestamp":"2025-09-28 20:09:40:940","userAgent":"PostmanRuntime/7.48.0"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Cannot read properties of undefined (reading 'databaseService')\u001b[39m","stack":"TypeError: Cannot read properties of undefined (reading 'databaseService')\n    at getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:19:42)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at validateUsername (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\routes\\userRoutes.js:17:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:284:15\n    at param (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:365:14)","timestamp":"2025-09-28 20:09:52:952"}
{"body":{},"ip":"::ffff:127.0.0.1","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Cannot read properties of undefined (reading 'databaseService')\u001b[39m","params":{},"query":{},"timestamp":"2025-09-28 20:09:52:952","userAgent":"PostmanRuntime/7.48.0"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:158:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-28 20:10:47:1047"}
{"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Cannot read properties of undefined (reading 'databaseService')\u001b[39m","stack":"TypeError: Cannot read properties of undefined (reading 'databaseService')\n    at getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:19:42)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at validateUsername (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\routes\\userRoutes.js:17:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:284:15\n    at param (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\router\\index.js:365:14)","timestamp":"2025-09-28 20:11:40:1140"}
{"body":{},"ip":"::1","level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Cannot read properties of undefined (reading 'databaseService')\u001b[39m","params":{},"query":{},"timestamp":"2025-09-28 20:11:40:1140","userAgent":"curl/8.12.1"}
{"context":"DatabaseService","isImmutableError":false,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in upsertUser:\u001b[39m","path":"instagram_id","timestamp":"2025-09-28 20:12:35:1235"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to scrape data for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:12:35:1235"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:12:35:1235"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Database error while saving user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-28 20:12:35:1235","userAgent":"curl/8.12.1"}
{"context":"DatabaseService","isImmutableError":false,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in upsertUser:\u001b[39m","path":"instagram_id","timestamp":"2025-09-28 20:13:15:1315"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to scrape data for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:13:15:1315"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:13:15:1315"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Database error while saving user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-28 20:13:15:1315","userAgent":"curl/8.12.1"}
{"context":"DatabaseService","isImmutableError":false,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in upsertUser:\u001b[39m","path":"instagram_id","timestamp":"2025-09-28 20:13:32:1332"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to scrape data for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:13:32:1332"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:13:32:1332"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Database error while saving user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-28 20:13:32:1332","userAgent":"curl/8.12.1"}
{"context":"DatabaseService","isImmutableError":false,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in upsertUser:\u001b[39m","path":"instagram_id","timestamp":"2025-09-28 20:14:43:1443"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mFailed to scrape data for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:14:43:1443"}
{"isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mError in getUser for tantrafiesta: Database error while saving user\u001b[39m","stack":"Error: Database error while saving user\n    at DatabaseService.upsertUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\services\\DatabaseService.js:212:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async UserController.scrapeAndStoreAllData (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:221:23)\n    at async UserController.getUser (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\src\\controllers\\UserController.js:44:7)","statusCode":500,"timestamp":"2025-09-28 20:14:43:1443"}
{"body":{},"ip":"::1","isOperational":true,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mGET /api/user/tantrafiesta - Database error while saving user\u001b[39m","params":{},"query":{},"statusCode":500,"timestamp":"2025-09-28 20:14:43:1443","userAgent":"curl/8.12.1"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:158:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-28 20:16:22:1622"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:158:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-28 20:17:49:1749"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:158:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-28 20:20:42:2042"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"\u001b[31merror\u001b[39m","message":"\u001b[31mServer error: listen EADDRINUSE: address already in use :::8000\u001b[39m","port":8000,"stack":"Error: listen EADDRINUSE: address already in use :::8000\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (D:\\Personal\\Projects\\PrimeSpot\\shreeBackend\\backend\\server.js:158:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-09-28 20:21:04:214"}
